src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserResource.kt

```diff
@@ -163,6 +163,20 @@ class UserResource(
     ) @RequestParam @NotEmpty lastName: String
   ): List<UserSummaryWithEmail> = userService.findUsersByFirstAndLastNames(firstName, lastName)
 
+  @PreAuthorize("hasRole('ROLE_MANAGE_NOMIS_USER_ACCOUNT')")
+  @GetMapping("/all")
```

Comment:
Not sure whether the naming of the endpoint should be **/users/active** ?

Reply:
`users/active` sounds reasonable